src/styles/blocks/card.scss

```diff
@@ -0,0 +1,59 @@
+.card {
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 33px;
+  }
+
+  &__title {
+    display: flex;
+    margin: 40px 0 0;
+    width: 166px;
+
+    font-weight: $font-weight--medium;
+    font-size: 12px;
+    line-height: 18px;
+    color: $c-dark;
+  }
+
+  &__art-number {
+    display: flex;
+    align-self: flex-start;
+    margin: 4px 0 0 17px;
```

Comment:
better not to use everywhere margin 17px, better make it as paddings for the card